extensions/amp-story/1.0/amp-story-draggable-drawer.css

```diff
@@ -91,3 +96,32 @@
   border-radius: 8px !important;
   overflow: hidden !important;
 }
+
+/** Bookend overrides. */
```

Comment:
Should these go here?  Doesn't this break encapsulation a little bit, that subclass styles are defined in the superclass?

Reply:
You're 100% right

The issue is that the `amp-story-bookend.css` code goes into the bookend shadow DOM. We could have this CSS block into the main `amp-story.css` file, or keep it in the `draggable-drawer.css` file. I picked the latter, but I don't feel strongly. What do you think?

Reply:
SGTM.  In theory we could add a new bookend-specific CSS that is the "outer" styles of the bookend, but that's just a lot of overhead to maintain, so... 🤷‍♂